opentelemetry/proto/common/v1/common.proto

```diff
@@ -53,3 +53,10 @@ message StringKeyValue {
   string key = 1;
   string value = 2;
 }
+
+// LibraryInfo is a message representing the instrumented library informations such as the fully
+// qualified name and version. 
+message LibraryInfo {
```

Comment:
language like here: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md#library ?

Also if we will want to distinguish library that instrumented code vs. SDK version that uploaded telemetry. Will SDK version be part of LibraryInfo or be part of a resource attributes?

Reply:
I expect that SDK version is part of `Resource` and follows `Library` section of semantic conventions doc that you linked to. Perhaps for clarity we need to rename the section to `Telemetry Library`. There is always one SDK per `Resource`.

There can be many _Instrumenting libraries_ (and corresponding Named Tracer/Meter) per `Resource` and this is what I think this PR suggests to represent differently.

Reply:
sorry, it was two comments in one. Splitting